Automattic/woocommerce-payments
# BNPLs: updated methods copy on settings page (part II)

Fixes #6799 

#### Changes proposed in this Pull Request

- Updated Affirm/Afterpay PMs description on the setting page. This is a new attempt to fix the issue after we realized the [previous implementation](https://github.com/Automattic/woocommerce-payments/pull/6880) had a flaw. Since it was already merged when we found out, I had to [revert](https://github.com/Automattic/woocommerce-payments/pull/6918) the changes. 
- We do not rely in the default currency anymore, as there can be cases where this currency will not match the country one in the Stripe account. Instead, we use the country in the Stripe account to calculate the currency that will be displayed.









#### Testing instructions





* Check settings page to ensure the copy is updated.
* Check JS console to ensure no JS errors are being introduced (incl. when checking/unchecking boxes)



-------------------


**Post merge**
